src/utils/formatters.ts

```diff
@@ -0,0 +1,58 @@
+import { isArray } from 'lodash'
+import { IMiddleware } from '../types/interfaces'
+
+function extractMiddlewareName(middlewareName: string): string {
+	const tempName = middlewareName.replace('<', '').replace('>', '')
+
+	return `${tempName.charAt(0).toUpperCase() + tempName.slice(1)}`
+}
+
+/**
+ * Works only with first middleware argument and prints his value if it consists of a primitive value array or an object containing such arrays
+ * @param {string} middlewareName
+ * @param {IMiddleware} middleware object containing all necessary information about actual middleware
+ * @return { string } middleware's description
+ * */
+export const basicArrayFormatter = (middlewareName: string, middleware: IMiddleware): string => {
+	const permissionsResult = `${extractMiddlewareName(middlewareName)}: `
```

Comment:
@kot-1999 middlewareName should be already formatted string so the extractMiddlewareName fn here should not be need